lib/buffer.js

```diff
@@ -933,8 +940,12 @@ Buffer.prototype.readUInt32BE = function(offset, noAssert) {
 
 
 Buffer.prototype.readIntLE = function(offset, byteLength, noAssert) {
+  if (!noAssert)
+    checkByteLength(byteLength);
```

Comment:
So should I check the type, or not? Do you think the type check will affect the performance?